src/modules/tracker.js

```diff
@@ -2235,6 +2235,391 @@ class Tracker {
+      // Ensure parameters are provided (required)
+      const baseUrl = `${this.behavioralV2Url}assistant_submit?`;
+      const {
+        section,
+        intent,
+      } = parameters;
+      const queryParams = {};
+      const bodyParams = {
+        intent,
+        section,
+      };
+
+      if (section) {
+        queryParams.section = section;
+      }
+
+      const requestURL = `${baseUrl}${applyParamsAsString(queryParams, this.options)}`;
+      const requestMethod = 'POST';
+      const requestBody = applyParams(bodyParams, {
+        ...this.options,
+        requestMethod,
+      });
+      this.requests.queue(
+        requestURL,
+        requestMethod,
+        requestBody,
+        networkParameters,
+      );
+      this.requests.send();
+      return true;
+    }
+
+    this.requests.send();
+
+    return new Error('parameters are required of type object');
+  }
+
+  /**
+   * Send assistant results page load started
+   *
+   * @function trackAssistantLoadStarted
+   * @param {object} parameters - Additional parameters to be sent with request
+   * @param {string} parameters.intent - Intent of of user request
+   * @param {string} [parameters.section] - The section name for the item Ex. "Products"
+   * @param {string} [parameters.intentResultId] - The intent result id from the ASA response
+   * @param {object} [networkParameters] - Parameters relevant to the network request
+   * @param {number} [networkParameters.timeout] - Request timeout (in milliseconds)
+   * @returns {(true|Error)}
+   * @description Assistant results page load begun (but has not necessarily loaded completely)
+   * @example
+   * constructorio.tracker.trackAssistantLoadStarted(
+   *     {
+   *         intent: 'show me a recipe for a cookie',
+   *     },
+   * );
+   */
+  trackAssistantLoadStarted(parameters, networkParameters = {}) {
+    if (parameters && typeof parameters === 'object' && !Array.isArray(parameters)) {
+      // Ensure parameters are provided (required)
+      const baseUrl = `${this.behavioralV2Url}assistant_result_load_start?`;
+      const {
+        section,
+        intentResultId,
+        intent,
+      } = parameters;
+      const bodyParams = {
+        intent_result_id: intentResultId,
+        intent,
+        section,
+      };
+
+      const requestURL = `${baseUrl}${applyParamsAsString({}, this.options)}`;
+      const requestMethod = 'POST';
+      const requestBody = applyParams(bodyParams, {
+        ...this.options,
+        requestMethod,
+      });
+      this.requests.queue(
+        requestURL,
+        requestMethod,
+        requestBody,
+        networkParameters,
+      );
+      this.requests.send();
+      return true;
+    }
+
+    this.requests.send();
+
+    return new Error('parameters are required of type object');
+  }
+
+  /**
+   * Send assistant results page load finished
+   *
+   * @function trackAssistantLoadFinished
```

Comment:
Same as above, we might want to add `result` in the name if we do it for the start function